proto/desmos/subspaces/v1beta1/subspace.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package desmos.subspaces.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/desmos-labs/desmos/x/staging/subspaces/types";
+
+// Subspace contains all the data of a Desmos subspace
+message Subspace {
```

Comment:
I think we should add another field to the subspace named like `open` or something similar, that allows to specify whether posting inside such subspace is allowed without any registration. This would allow users to create open communities that do not require subspace owners to manually approve people to post inside them

Reply:
wouldn't be simpler if we let subspaces opened by default and then let admin/owner to block someone if they're not respecting the rules?

Reply:
@bragaz No, I think it's better to have subspaces closed by default, optionally open to everyone, and allow admins to both: 

- register new users
- block existing users

This would be an implementation that's more similar to current social networks, where by default you are not registered and you have to go through a custom procedure to be allowed to use them. If we set subspaces open by default, developers would not be able to do this cause they would have to block all existing users and then unblock them later. It's much easier if we: 

1. implement an `Open` field that tells that subspace does not require a registration 
2. allow subspace admins to allow individual Desmos accounts to post inside it

Reply:
Ok gotcha!